voidcrew/code/modules/research/server.dm

```diff
@@ -0,0 +1,16 @@
+/obj/machinery/rnd/server/multitool_act(obj/item/multitool, mob/user, params)
+	if(istype(multitool, /obj/item/multitool))
```

Comment:
you don't need to check this, multitool_act already handles it

Reply:
For some reason removing this causes the runtime `Runtime(\"Undefined variable /mob/living/carbon/human/var/buffer\\n\")`

Reply:
> For some reason removing this causes the runtime `Runtime(\"Undefined variable /mob/living/carbon/human/var/buffer\\n\")`

ignore, mda didnt pull